pkg/envinfo/envinfo.go

```diff
@@ -150,17 +152,14 @@ func newProxyEnvInfo() proxyEnvInfo {
 func (esi *EnvSpecificInfo) SetConfiguration(parameter string, value interface{}) (err error) {
 	if parameter, ok := asLocallySupportedParameter(parameter); ok {
 		switch parameter {
-		case "create":
```

Comment:
Why are we removing the switch the create here?

Reply:
@johnmcollier That's because he switches to use `co.EnvSpecificInfo.SetComponentSettings` to set up the parameters for `odo create` below instead of `co.EnvSpecificInfo.SetConfiguration`

Reply:
@girishramnani Can we switch to use `co.EnvSpecificInfo.SetComponentSettings` as well for `odo url` given the argument of that function is `ComponentSettings` struct which includes all metadata of `odo url`, so that we can make the parameter setup consistent rather than using separate function for parameter setup.

Reply:
actually we have in the past preferred to be targeted, so if you are changing/adding a url then use the interface of url. so would it be ok if we keep that as is.

Reply:
I think all odo commands share the same mechanism (`envinfo.go`) to configure the `env.yaml` file so it would be better to centralize and make the mechanism generic, but it's fine, we can keep as it is for future url change.